Manifest.toml

```diff
@@ -232,9 +238,9 @@ version = "1.1.0"
 
 [[RegistryTools]]
 deps = ["AutoHashEquals", "LibGit2", "Pkg", "UUIDs"]
-git-tree-sha1 = "8c48f34b5884366b2b9e76311b38b2a6235f9095"
+git-tree-sha1 = "3dfa671318ac8af835cc621b08fcdcc28a4aee67"
 uuid = "d1eb7eb1-105f-429d-abf5-b0f65cb9e2c4"
-version = "1.3.1"
+version = "1.3.2"
```

Comment:
While we are here, can we actually require v1.3.4 in Project.toml and update the Manifest.toml accordingly?  See https://github.com/JuliaPackaging/Yggdrasil/pull/777